build/repo.targets

```diff
@@ -154,22 +122,26 @@
       <Solution>
         <AdditionalProperties>RestoreGraphOutputPath=$(_RestoreGraphSpecsDirectory)%(Solution.Repository)\%(Solution.FileName)%(Solution.Extension).json</AdditionalProperties>
       </Solution>
+
+      <GraphSpecInputs Include="
```

Comment:
Seems okay enough for now as long as these cached files wouldn't cause a build to pass that shouldn't. If there are any changes in the repo-to-repo graph, a bad cache will fail to build, right?